libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -273,13 +273,13 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
                 predefinedEventProperties.put("menu_item", "site_settings");
                 break;
             case OPENED_ACCOUNT_SETTINGS:
-                eventName = "me_opened_account_settings";
+                eventName = "me_account_settings_opened";
```

Comment:
Sure I can. The reason I've added the `me_` prefix was most of the other events have that kind of prefix to identify where the event is related to. I guess these ones are generic enough that we don't need. Will wait for confirmation from @daniloercoli and drop it if he agrees. Thanks!